### ISSUE TITLE
No calls to OnXXXChanged event handlers on Avalonia.StyledProperty

### ISSUE THREAD

**Issue by graealex** [HavenDV/DependencyPropertyGenerator#12]
### Describe the bug

When creating a `StyledProperty` via for example:

`[DependencyProperty<int>("SelectedType")]`

The resulting generated code does never call the `OnSelectedTypeChanged(...)` event handler, subsequently no event notifications when the value changes are received.

The same property as a `DirectProperty`:

`[DependencyProperty<int>("SelectedType", IsDirect = true)]`

contains the necessary calls on all versions of the `OnSelectedTypeChanged(...)` event handler in the property setter.

I understand that event handlers for value changes have become a bit cumbersome, as the `notifying` parameter has become internal (as of Avalonia 11 preview 6), and as such is not available anymore. The preferred route to receive event notifications is to either override the `OnPropertyChanged` method, or attach to the `PropertyChange` event. (as per Max Katz from AvaloniaUI core developer team)

### Steps to reproduce the bug

Create `StyledProperty` in an Avalonia project and try to listen for value changes, which will not fire.

### NuGet package version

1.2.5

### IDE

Visual Studio 2022

**Comment by HavenDV** [comment]
```
//HintName: MyControl.StaticConstructor.generated.cs

using System;

#nullable enable

namespace H.Generators.IntegrationTests
{
    public partial class MyControl
    {
        static MyControl()
        {
            IsSpinningProperty.Changed.Subscribe(static x =>
            {
                ((global::H.Generators.IntegrationTests.MyControl)x.Sender).OnIsSpinningChanged(
                    (bool)x.OldValue.GetValueOrDefault(),
                    (bool)x.NewValue.GetValueOrDefault());
            });
        }
    }
}
```

In addition to the main properties file for Avalonia, a static constructor is also generated that calls the OnChanged methods. Please let me know if there are any problems with this.

**Comment by HavenDV** [comment]
Yes, I added integration tests for this case and it doesn't really call the methods. I will research this

**Comment by graealex** [comment]
Thanks for the fast response - Preview 5 and 6 broke a lot of things. For now the workaround is to manually override `OnPropertyChanged`, but that's certainly not particularly elegant. Or maybe they unintentionally broke the `Subscribe` method as well.

**Comment by HavenDV** [comment]
Yes, they changed the Subscribe method and now it does not contain data about the sender, old and new value. But the option via overriding OnPropertyChanged will not work for attached properties.
So I think that now it cannot be bypassed and we need to wait for a stable version. I rechecked and everything definitely works for stable versions of Avalonia
